compilers/esm.js

```diff
@@ -101,7 +101,7 @@ exports.compile = function(load, opts, loader) {
       lowResolutionSourceMap: opts.lowResSourceMaps
     });
 
-    var tree = load.metadata.parseTree || compiler.parse(load.source, load.path);
+    var tree = load.metadata.parseTree || compiler.parse(source, load.path);
```

Comment:
scroll up!
https://github.com/AdamBurgess/builder/blob/fix-esm-plugins/compilers/esm.js#L135-L136
it was added in f0236 but then later on was forgotten :(